auto_nag/scripts/survey_sec_bugs.py

```diff
@@ -0,0 +1,100 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag.bzcleaner import BzCleaner
+
+
+class SurveySecurityBugs(BzCleaner):
+    LIST_OF_PEOPLE_TO_REACH_OUT = [
+        "continuation@gmail.com",  # mccr8
+        "jdemooij@mozilla.com",  # jandem (js/jit)
+        "nical.bugzilla@gmail.com",  # nical (gfx)
+        "emilio@crisal.io",  # emilio (css/style/layout)
+    ]
+
+    def __init__(self):
+        super(SurveySecurityBugs, self).__init__()
+        self.changes_per_bug = {}
+
+    def description(self):
+        return "Submit survey to assignee of a security bug"
+
+    def get_bz_params(self, date):
+        params = {
```

Comment:
nit: sort the f,o,v fields, I'd put chfield... after resolution and then f1,o1,v1,f2,o2,v2,....

Reply:
Would you mind if I leave them unsorted? Sorting them like field/operator/value makes the query readable in a way that the query is reviewable IMHO.